inference-engine/src/gna_plugin/descriptions/gna_input_desc.cpp

```diff
@@ -14,11 +14,15 @@
 using namespace InferenceEngine;
 using namespace GNAPluginNS;
 
-size_t InputDesc::minBytesRequiredForStoreInput(CNNLayerPtr layer) {
+size_t InputDesc::minBytesRequiredForStoreInput(CNNLayerPtr layer, const bool low_precision) {
     auto quantized = getInjectedData<QuantizedLayerParams>(layer);
     size_t precision_bytes;
     if (quantized) {
-        precision_bytes = 2;
+        if (low_precision) {
```

Comment:
Done. It might be worth considering in future to refactor some other parts of code using QuantizedLayerParam (e.g. scale factor calulator) to take advantage of this change.